src/dlgAboutDialog.cpp

```diff
@@ -940,12 +938,13 @@ void dlgAboutDialog::setSupportersTab(const QString& htmlHead)
     }
 
     QString supporters_text(QStringLiteral(R"(
-               <p><div style="text-align: center">%1</div></p>
+               <p><br><div style="text-align: center">%1<br></div></p>
                %2
+               <p><br><div style="text-align: center">%3<br></div></p>
                )")
                   .arg(tr(R"(
-                          These formidable folks will be fondly remembered forever<br>for their generous sponsorship on <a href="https://www.patreon.com/mudlet">Mudlet's patreon</a>:
-                          )"), supporters_image_html));
+                          These formidable folks will be fondly remembered forever<br>for their generous financial support on <a href="https://www.patreon.com/mudlet">Mudlet's patreon</a>:
+                          )"), supporters_image_html, tr("Go to <a href=\"https://www.patreon.com/mudlet\">patreon.com/mudlet</a> if you'd like to be mentioned here.")));
```

Comment:
The tooltip is kind of degrading in the obviousness of the message it delivers - anyone can figure out that you can go there to be mentioned.